test/smoke/src/areas/positron/test-explorer/test-explorer.test.ts

```diff
@@ -0,0 +1,125 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2024 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+
+import path = require('path');
+import { Application, Logger, PositronRFixtures, PositronUserSettingsFixtures } from '../../../../../automation';
+import { installAllHandlers } from '../../../utils';
+import { expect } from '@playwright/test';
+
+export function setup(logger: Logger) {
+	describe('Test Explorer', () => {
+		// Shared before/after handling
+		installAllHandlers(logger);
+
+		let app: Application;
+		let userSettings: PositronUserSettingsFixtures;
+
+		describe('Test Explorer - R', () => {
+			before(async function () {
+				app = this.app as Application;
+
+				try {
+
+					await PositronRFixtures.SetupFixtures(this.app as Application);
+
+					userSettings = new PositronUserSettingsFixtures(app);
+
+					// don't use native file picker
+					await userSettings.setUserSetting([
+						'files.simpleDialog.enable',
+						'true',
+					]);
```

Comment:
I'm just curious. What's this about? From the outside, it would seem unrelated.

Reply:
Good question.  That eliminates the native file picker dialog so the test is not OS dependent.  Once that setting is set, a VSCode file picker is available instead.

Reply:
And file picking comes up ... when opening the folder that is a package that we use as the test fixture?

Reply:
Yeah, exactly.  The test opens with the qa-example-content directory, and then I need to navigate down to the subdirectory to make Positron think its an R package.